### DISCUSSION TITLE
Problems with no interuptions channels

### DISCUSSION THREAD

**Discussion by computertech1012** [BearWare/TeamTalk5#2145] [General]
Hi,
Hello everyone! I will do my best to explain. there is a bug with no interuptions channels. The default timer is not working correctly.
If you create a channel with no interuptions and set the delay before next user can transmit to 0, it doesn't allow any transmission. The default timer is set to 500. This is a problem because you go to update an existing channel to this setting, and you don't want to have this feature enabled, setting it to 0 causes problems. So my question is why is the default timer set to 500?
If you create a new channel and set it up to no interuptions, and join the channel by default, it will have your default timer for the next user set to 500 even though this doesn't actually happen.
I think it need to be either clarified what's wrong with this feature, or what's going on here. Because for example if I have a channel set to voice everyone can talk, and then later I go to set it up so that no users can interupt each other, but I don't want to put a time frame on that, so I set it to 0 or just leave it alone, it's causing issues. Or, if you ar creating a new channel like this it's also giving me errors without messages, like not allowing transmissions. If I go set the time to 500 then it works fine but doesn't actually do anything.
I hope you understand the problem, why is the default time 500 and not 0? Why does 500ms do nothing? Shouldn't it be 0 by default?
Maybe I am not understanding so please let us know. TeamTalk Client and server 5.15.0.5135.
Thanks for reading!

**Comment by bear101** [comment]
The server makes the switch to a new user when nothing has been received from the "head" of the transmitters. If the server should switch after 0 msec then the server would immediately switch to the next one.

In general you should not specify a switch timeout that is close to the duration of your audio packet's content (default 40 msec).

**Comment by bear101** [comment]
If your audio packets contain 100 msec of audio and your transmit switch is 50 msec then the server would switch to another user before the current "head" had time to transmit the next audio packet.

**Reply by computertech1012** [reply]
Hi,
@bear101 that makes sence now thanks for clarifying.
So could you make it so that especially when you create a channel or update a channel with your android, it will set it to 500? Because as of now if you don't change any settings, and you change a channel that is at the moment not set to no interuptions, and you set it that way, then it's default set to 0 and if you are on android or iOS you can't change this to 500. I think the default should be changed to 500 when updating the channels as well.
Thanks for reading!
